manually_run-test_all_samples.py

```diff
@@ -0,0 +1,137 @@
+import os
```

Comment:
is `manually_run-test-all-samples.py` in pytests suite, or external standalone script?

Reply:
External, so that it's run manually